VERSION

```diff
@@ -82,17 +84,17 @@ date="Unreleased developer copy"
 # Version numbers are described in the Libtool current:revision:age
 # format.
 
-libmpi_so_version=30:0:10
+libmpi_so_version=31:0:11
```

Comment:
Ok, I guess I shouldn't be using `--no-merge` in my `git log`.

So I now see fda91572ce that modified `mpi.h`.  But it just added a `#define` -- it didn't add any interfaces.  So I don't think we should be bumping to `31:0:11`, but rather to `30:1:10`.